components/database/postgresql-10-mysql_fdw/Makefile

```diff
@@ -43,9 +41,9 @@ TEST_TARGET=	$(NO_TESTS)
 
 include $(WS_MAKE_RULES)/common.mk
 
-COMPONENT_BUILD_ENV	+=	USE_PGXS=0
+COMPONENT_BUILD_ENV	+=	USE_PGXS=1
 COMPONENT_BUILD_ARGS	+=	MYSQL_LIBNAME=$(MYSQL_LIBDIR)/libmysqlclient.so
-COMPONENT_INSTALL_ENV	+=	USE_PGXS=0
+COMPONENT_INSTALL_ENV	+=	USE_PGXS=1
 
-REQUIRED_PACKAGES += $(PG_DEVELOPER_PKG)
+REQUIRED_PACKAGES += database/postgres-10/developer
 REQUIRED_PACKAGES += database/mariadb-101/library
```

Comment:
We probably should drop postgresql-96 before this.

Reply:
Yeah 96 is not longer supported. But it seems on the build server all devel versions of previous pg versions are installed because on my clean server all older versions fail to build to not being able to find pg_config for the right version.